ballerina/Ballerina.toml

```
@@ -1,16 +1,19 @@
 [package]
 org = "ballerinax"
 name = "snowflake"
-version = "1.2.0"
+version = "1.3.0"
```

I think we can do a patch release with this
```suggestion
version = "1.2.1"
```